app/templates/records/record_detail.html

```diff
@@ -37,13 +37,45 @@ <h1 class="tna-hgroup__title">{{ pageTitle }}</h1>
 
   {{ information_boxes(record) }}
 
-  {{ detail_fields(record, field_labels, request) }}
+<div class="tna-container tna-!--margin-top-l">
+  <div class="tna-column tna-column--full">
 
-  {{ how_to_order() }}
-
-  {{ series_information(record) }}
+    <h3 class="tna-accordion__heading tna-heading-l">
+      Full description and record details
+    </h3>
+    <div class="tna-accordion__body" id="-content-1">
```

Comment:
All these IDs need to be set.

This could be `record-details` for example.

Reply:
ID has now been set